src/styles/components/_component-card.scss

```diff
@@ -28,4 +26,10 @@
     display: block;
     border-top: 0.25rem solid black;
   }
+
+  :focus {
+    outline: none;
+    border: 2px solid map.get($colors, "accent-teal");
+  }
 }
+
```

Comment:
I think all of these `styles/*` changes are unrelated to this PR. Was this one of those other fixes that snuck in here? I would expect that no files from the `styles` folder would end up in this PR.

Reply:
Honestly, I think these changes should have already been in here, but with dropping those fix/style commits with the changes I made, I broke the stylesheet that was originally here. I can remove this commit if you want (I just gotta make sure I merged the branches in the right order to get this code back!)